packages/components/tests/integration/components/hds/alert/index-test.js

```diff
@@ -0,0 +1,216 @@
+    // here we don't test all the possible combinations, only some of them as precaution
+    assert.expect(6);
+    await render(hbs`<Hds::Alert @type="inline" @description="yo" />`);
+    assert.dom('.flight-icon-info').exists();
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="yo" @type="compact" />`
+    );
+    assert.dom('.flight-icon-info-fill').exists();
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="yo" @color="highlight" />`
+    );
+    assert.dom('.flight-icon-info').exists();
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="yo" @color="success" />`
+    );
+    assert.dom('.flight-icon-check-circle').exists();
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="yo" @color="warning" />`
+    );
+    assert.dom('.flight-icon-alert-triangle').exists();
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="yo" @color="critical" />`
+    );
+    assert.dom('.flight-icon-alert-diamond').exists();
+  });
+
+  test('if an icon is declared, the icon should render in the component and override the default one', async function (assert) {
+    assert.expect(2);
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="yo" @icon="clipboard-copy" />`
+    );
+    assert.dom('.flight-icon-clipboard-copy').exists();
+    await render(
+      hbs`<Hds::Alert @description="yo" @type="compact" @icon="clipboard-copy" />`
+    );
+    assert.dom('.flight-icon-clipboard-copy').exists();
+  });
+
+  test('it should display no icon when @icon is set to false', async function (assert) {
+    await render(
+      hbs`<Hds::Alert @type="inline" @title="yo" @icon={{false}} />`
+    );
+    assert.dom('.flight-icon').doesNotExist();
+  });
+
+  // TEXT (TITLE + DESCRIPTION)
+
+  test('it should render the title when the @title argument is provided', async function (assert) {
+    await render(hbs`<Hds::Alert @type="inline" @title="This is the title" />`);
+    assert.dom(this.element).hasText('This is the title');
+  });
+  test('it should render the description when the @description argument is provided', async function (assert) {
+    await render(
+      hbs`<Hds::Alert @type="inline" @description="This is the description" />`
+    );
+    assert.dom(this.element).hasText('This is the description');
+  });
+  test('it should render both the title and the description when both the @title and @description arguments are provided', async function (assert) {
+    assert.expect(2);
+    await render(
+      hbs`<Hds::Alert @type="inline" @title="This is the title" @description="This is the description" />`
+    );
+    assert.dom('.hds-alert__title').hasText('This is the title');
+    assert.dom('.hds-alert__description').hasText('This is the description');
+  });
+  test('it should render rich HTML when the @description argument contains HTML tags', async function (assert) {
```

Comment:
[comment] I'm surprised that we are even offering this.

Reply:
> [comment] I'm surprised that we are even offering this.
we have multiple of this cases [in the audit](https://www.figma.com/file/h3LtloARmbYVwh7z6XWHVb/?node-id=3%3A4), we have to support them. and we're (more correctly, we'll be) supporting 
basic HTML elements like `a`, `strong`, `em`, `code` ([see CSS and comments](https://github.com/hashicorp/design-system/blob/b3c9b29a5c500830a9d97dbbb3a598d2d490d125/packages/components/app/styles/components/alert.scss#L65-L88))